ovs/dal/tests/test_basic.py

```diff
@@ -1339,17 +1339,27 @@ def test_delete_retries(self):
         Validates whether a delete will retry on an assert error
         """
         def _change_list():
-            if Basic._executed is False:
-                PersistentFactory.store.set('ovs_listcache_testdisk', {})
-                Basic._executed = True
+            # Trigger an AssertException on ovs_unique constraint
+            unique_key = 'ovs_unique_{0}_{{0}}_{{1}}'.format(disk._classname)
+            store_data = disk._persistent.get(disk._key)
+            for prop in disk._properties:
+                if prop.unique is True:
+                    if prop.property_type not in [str, int, float, long]:
```

Comment:
I suggest you define this list as '''allowed_types''', and join it in the error message. 
-> consistent error messaging if the list ever needs to be changed